### ISSUE TITLE
RCall with custom R installation from source

### ISSUE THREAD

**Issue by cth** [JuliaInterop/RCall.jl#156]
I've installed R 3.3.2 using an configuration like the one below: 

`./configure --prefix=/home/fng514/local --enable-fast-install=no --enable-R-shlib=yes --with-pic`

It works fine when I start R from the command-line, but RCall runs into some problems with the R dyn.load function when it tries to initialize some libraries on startup. 

```
              _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.5.0 (2016-09-19 18:14 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-pc-linux-gnu

julia> ENV["R_HOME"]
"/home/fng514/local/lib64/R"

julia> using RCall
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/fng514/local/lib64/R/library/methods/libs/methods.so':
  /home/fng514/local/lib64/R/library/methods/libs/methods.so: undefined symbol: R_dot_Generic
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/fng514/local/lib64/R/library/utils/libs/utils.so':
  /home/fng514/local/lib64/R/library/utils/libs/utils.so: undefined symbol: R_CStackDir
In addition: Warning message:
package "methods" in options("defaultPackages") was not found 
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/fng514/local/lib64/R/library/grDevices/libs/grDevices.so':
  /home/fng514/local/lib64/R/library/grDevices/libs/grDevices.so: undefined symbol: do_bmVersion
In addition: Warning message:
package ‘utils’ in options("defaultPackages") was not found 
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/fng514/local/lib64/R/library/grDevices/libs/grDevices.so':
  /home/fng514/local/lib64/R/library/grDevices/libs/grDevices.so: undefined symbol: do_bmVersion
In addition: Warning message:
package ‘grDevices’ in options("defaultPackages") was not found 
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/fng514/local/lib64/R/library/grDevices/libs/grDevices.so':
  /home/fng514/local/lib64/R/library/grDevices/libs/grDevices.so: undefined symbol: do_bmVersion
In addition: Warning message:
package ‘graphics’ in options("defaultPackages") was not found 
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/fng514/local/lib64/R/library/methods/libs/methods.so':
  /home/fng514/local/lib64/R/library/methods/libs/methods.so: undefined symbol: R_dot_Generic
In addition: Warning message:
package ‘stats’ in options("defaultPackages") was not found 
During startup - Warning message:
package ‘methods’ in options("defaultPackages") was not found 
```

It seems dyn.load works fine when I run R out of the box, but not when I run it through RCall. Any ideas why this is happening?

**Comment by simonbyrne** [comment]
Can you try setting LD_LIBRARY_PATH?: 
http://juliainterop.github.io/RCall.jl/latest/installation.html

**Comment by cth** [comment]
`export LD_LIBRARY_PATH="/home/fng514/local/lib64/R/lib"`  does the trick. I thought I tried that before, but seems I should have paid more attention the installation manual! Sorry. Anyway, happy that it works :-)

**Comment by simonbyrne** [comment]
No worries. The error message was supposed to be more informative, but there was a mistake in the logic: this is now fixed in 22911b10a7f4a3fa70ad426e7b7aa604d1bfdfa2.
